gradle-diff-task-dependencies/plugin/src/main/groovy/com/novoda/gradle/diffdependencytask/Constants.groovy

```diff
@@ -0,0 +1,6 @@
+package com.novoda.gradle.diffdependencytask
+
+class Constants {
+    public static String TASK_CONFIG_PROPERTY_NAME = 'diffDependency'
```

Comment:
If you would manage to remove the `static` block in `ConditionalDependency` you could get rid of this altogether